tests/unit/util.py

```diff
@@ -20,8 +20,8 @@ def bf16_required_version_check():
 
     if (TORCH_MAJOR > 1 or
         (TORCH_MAJOR == 1 and TORCH_MINOR >= 10)) and (CUDA_MAJOR >= 11) and (
-            NCCL_MAJOR > 2 or
-            (NCCL_MAJOR == 2 and NCCL_MINOR >= 10)) and torch_info['bf16_support']:
+            NCCL_MAJOR > 2 or (NCCL_MAJOR == 2 and NCCL_MINOR >= 10)
+        ) and torch.cuda.is_available() and torch.cuda.is_bf16_supported():
```

Comment:
Having a call to `torch.cuda.is_bf16_supported()` causes a problem with our unit tests (specifically the `distributed_test` decorator) because this function call initializes CUDA